### PR TITLE
Fix linting violations

### PR DESCRIPTION
**Pull request by fernflower**
used-before-assignment is a nice check worth keeping, so let's
fix commands.upgrade.util code.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py [hunk 1]
```diff
@@ -93,21 +93,30 @@ def include_directive_check(read_func=_read_file):
     included_files = ['/etc/cups/cupsd.conf']
     error_list = []
 
-    for included_file in included_files:
+    vetted_included_files = []
+    while included_files:
+        # NOTE(ivasilev) Will be using stack to process last encountered include directives first
+        included_file = included_files.pop(-1)
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py:99]:
```suggestion
        for included_file in included_files.copy():
```
?

Comment (fernflower) [reply repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py:99]: That is not the only place, another violation is in remove part. I felt pretty bad we missed that in original pr so I decided to refactor the implementation completely (as the functionality is decently covered by tests I can be sure my version has the same behaviour). Too much in your opinion? You'd rather leave it be?

Comment (vinzenz) [reply repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py:99]: I just realized you're right, I didn't get the whole context.

File: repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py [hunk 2]
```diff
@@ -93,21 +93,30 @@ def include_directive_check(read_func=_read_file):
...
+        includes_to_process = []
         for line in lines:
             value = get_directive_value('Include', line)
             if value:
-                included_files.append(value)
+                includes_to_process.append(value)
+        # NOTE(ivasilev) Add discovered Include directives to the stack in reversed order, so that they are processed
+        # in the same order they appeared in the file
+        includes_to_process.reverse()
+        included_files.extend(includes_to_process)
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/actors/cupsscanner/libraries/cupsscanner.py:117]:
```suggestion
        included_files.extend(reversed(includes_to_process))
```

**Review by pirat89 — APPROVED** [review]

**Pull request merged**